site/specs/webrtc.yml

```diff
@@ -2421,6 +2520,201 @@ components:
           example: participant1
         deviceApiVersion:
           $ref: '#/components/schemas/deviceApiVersionEnum'
+    createDialogRequest:
+      type: object
+      description: The important things to specify if for the success of the dialog
+      properties:
+        dialogType:
+          $ref: '#/components/schemas/dialogType'
+        dialogStateData:
+          oneOf:
+            - $ref: '#/components/schemas/sipDialogRequestStateData'
+    createDialogResponse:
+      type: object
+      description: a Dialog is used for exchanging user-level state information
```

Comment:
Looking at the rest of our API docs, the description starts with a capital letter. Good to make it consistent

Reply:
tried to improve. 
'a Dialog is used for exchanging high-level state information, the types of states and events that might
        be of interest to the users of the system - thing like "the other end answered the call"'